pillar/heroku/xpro.sls

```diff
@@ -26,6 +29,8 @@
 } %}
 {% set env_data = env_dict[environment] %}
 {% set business_unit = 'mitxpro' %}
+{% set pg_creds = salt.vault.cached_read('postgres-{env}-odlvideo/creds/odlvideo'.format(env=ENVIRONMENT)) %}
```

Comment:
For the cached_read you'll want to pass in a cache_prefix https://github.com/mitodl/hashicorp-vault-formula/blob/master/_modules/vault.py#L160